runtime/syntax/lua.yaml

```diff
@@ -73,8 +75,6 @@ rules:
         rules:
             - constant.specialChar: "\\\\."
 
-    - special: "\\\\[0-7][0-7][0-7]|\\\\x[0-9a-fA-F][0-9a-fA-F]|\\\\[abefnrs]|(\\\\c|\\\\C-|\\\\M-|\\\\M-\\\\C-)."
```

Comment:
This special rule was completely broken and unrelated to Lua. I have thus removed it.